src/observers.test.ts

```diff
@@ -318,7 +318,7 @@ describe('getEpochBoundariesForHeight', () => {
 });
 
 describe('getEntropyForEpoch', () => {
-  beforeEach(() => {
+  beforeAll(() => {
```

Comment:
would this stub get reset in the afterEach below?

Reply:
yep, thanks for catching. moved it to `afterAll` in [67a5a68](https://github.com/ar-io/testnet-contract/pull/165/commits/67a5a689be84831662d096dd9a1fcb7201efb954)